KHLY2700/KHLY2700-IOC-01App/Db/KDVM-common.db

```diff
@@ -0,0 +1,707 @@
+  field(DTYP, "stream")
+  field(EGU, "")
+  field(INP, "@devKeithley_2700.proto getResistanceDigits($(P)CHANNEL_END) $(PORT)")
+  field(SCAN, "1 second")
+  field(SIML, "$(P)SIM")
+  field(SIOL, "$(P)SIM:DIGIT_RANGE")
+  field(SDIS, "$(P)DISABLE")
+  info(INTEREST, "HIGH")
+
+}
+
+
+# ## Read the status value 3.
+record(ai, "$(P)CHANNEL_END") 
+{
+  field(DESC, "DIGIT value")
+  field(SCAN, "1 second")
+
+}
+
+record(bi, "$(P)AUTO_RANGE") 
+{
+    field(DESC, "Query Auto Range")
+    field(SCAN, "1 second")
+    field(DTYP, "stream")
+    field(INP,  "@devKeithley_2700.proto getAutoRange($(P)CHANNEL_END) $(PORT)")
+    field(ZNAM, "ON")
+    field(ONAM, "OFF")
+    field(SIML, "$(P)SIM")
+    field(SIOL, "$(P)SIM:AUTO_RANGE")
+    field(SDIS, "$(P)DISABLE")
+    info(INTEREST, "HIGH")
+
+}
+
+record(bo, "$(P)AUTO_RANGE:SP") 
+{
+    field(DESC, "Enable or Disable Range")
+    field(SCAN, "Passive")
+    field(DTYP, "stream")
+    field(OUT,  "@devKeithley_2700.proto setAutoRange($(P)CHANNEL_END) $(PORT)")
+    field(ZNAM, "ON")
+    field(ONAM, "OFF")
+    field(SIML, "$(P)SIM")
+    field(SIOL, "$(P)SIM:AUTO_RANGE:SP")
+    field(SDIS, "$(P)DISABLE")
+    info(INTEREST, "HIGH")
+
+}
+
+record(stringin, "$(P)BUFFER_STATS")
+{
+    field(DESC, "Available and reserved buffer size")
+    field(DTYP, "stream")
+    field(INP,  "@devKeithley_2700.proto getBufferStats $(PORT)")
+    field(PINI, "YES")
+    field(SIML, "$(P)SIM")
+    field(SIOL, "$(P)SIM:BUFFER_STATS")
+    field(SDIS, "$(P)DISABLE")
+}
+
+record(ai, "$(P)BUFFER_LOC") 
+{
+    field(DESC, "Next Buffer Location")
+    field(SCAN, "1 second")
+    field(DTYP, "stream")
+    field(INP,  "@devKeithley_2700.proto getNextBufferReading $(PORT)")
+    field(SIML, "$(P)SIM")
+    field(SIOL, "$(P)SIM:BUFFER_LOC")
+    field(SDIS, "$(P)DISABLE")
+}
+
+record(stringin, "$(P)LIST_OF_ELEMENTS")
+{
+    field(DESC, "DATA")
+    field(DTYP, "stream")
+    field(INP,  "@devKeithley_2700.proto getDataElements $(PORT)")
+    field(PINI, "YES")
+}
+
+record(stringout, "$(P)LIST_OF_ELEMENTS:SP")
+{
+    field(DESC, "DATA:SP")
+    field(DTYP, "stream")
+    field(SCAN, "1 second")
+    field(OUT,  "@devKeithley_2700.proto setDataElements $(PORT)")
+    field(PINI, "YES")
+}
+
+record(stringin, "$(P)LIST_OF_READINGS")
+{
+    field(DESC, "Buffer Readings")
+    field(DTYP, "stream")
+    field(INP,  "@devKeithley_2700.proto getReadingsRange($(P)START,$(P)COUNT) $(PORT)")
+    field(SCAN, "1 second")
+    field(SIML, "$(P)SIM")
+    field(SIOL, "$(P)SIM:LIST_OF_READINGS")
+    field(SDIS, "$(P)DISABLE")
+}
+
+record(ai, "$(P)START") 
+{
+    field(DESC, "DATA ELEMENT READ PARAMETER")
+    field(DTYP, "stream")
+    field(FLNK, "$(P)LIST_OF_READINGS")
+
+}
+
+record(ai, "$(P)COUNT") 
+{
+    field(DESC, "DATA ELEMENT READ PARAMETER")
+    field(DTYP, "stream")
+    field(FLNK, "$(P)LIST_OF_READINGS")
+
+
+}
+
+record(ai, "$(P)MEAS:VOLT:DC")
+{
+    field(DESC, "Configure Channels to DCV")
```

Comment:
I'm confused by this. The PV naming suggests it's a read back but this description suggests it's a set point. Please be clear.